docs/fusion.md

```diff
@@ -212,3 +212,8 @@ The following configuration options are available:
 
 `fusion.tagsPattern`
 : The pattern that determines how tags are applied to files created via the Fusion client (default: `[.command.*|.exitcode|.fusion.*](nextflow.io/metadata=true),[*](nextflow.io/temporary=true)`)
+
+`fusion.cacheSize`
+: :::{versionadded} 23.11.0-edge
+  :::
+: The maximum size of the local cache used by the Fusion client.
```

Comment:
it would be nice to keep these options in alphabetical order, even though the Fusion options aren't perfectly ordered at the moment

Reply:
I would say, keep `enabled` first but then alphabetize the rest

Reply:
done